pawnShop/src/main/java/com/example/pawnShop/Service/Contract/ProductTypeService.java

```diff
@@ -0,0 +1,14 @@
+package com.example.pawnShop.Service.Contract;
+
+import com.example.pawnShop.Dto.Product.ProductTypeDto;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface ProductTypeService {
+    ProductTypeDto getProductTypeById(UUID id);
```

Comment:
good one again, you got the idea how interfaces should be extended & implemented ;)